### ISSUE TITLE
Ring Down and wont install anymore

### ISSUE THREAD

**Issue by macjeff** [UniversalDevicesInc/ring-nodeserver#6]
I noticed my Ring nodeserver showed FALSE inside the ISY.   Tried everything but could not get it to connect.

It kept saying COULD NOT INSTALL PROFILE INTO THE ISY on the PGC screen

So I deleted it totally from PGC.  It successfully deleted from the ISY just fine.

(I know it was connecting because I literally watched the nodes disappear from the ISY when I deleted from PGC)

Then I added it back in PGC but it did not come into the ISY. 

Rebooted ISY and cant get it to work.

It had been working fine.

If I go to the LOG for Ring in PGC it is empty but I get this error

startLogStream error :: Error: Internal error occurred: Authorization error (user=kubelet-api, verb=get, resource=nodes, subresource=proxy) at _request (/app/node_modules/kubernetes-client/backends/request/client.js:189:25) at Request.request [as _callback] (/app/node_modules/kubernetes-client/backends/request/client.js:148:14) at Request.self.callback (/app/node_modules/request/request.js:185:22) at Request.emit (events.js:197:13) at Request.EventEmitter.emit (domain.js:439:20) at Request. (/app/node_modules/request/request.js:1161:10) at Request.emit (events.js:197:13) at Request.EventEmitter.emit (domain.js:439:20) at IncomingMessage. (/app/node_modules/request/request.js:1083:12) at Object.onceWrapper (events.js:285:13)

** I did notice when I add it to PGC if I go under NodeServers menu its not there but if I go down to configure it is there.

I also tried another slot and still wont show up.  It just says DISCONNECTED on the PGC but if I try and start it, I get message its already started

**Comment by macjeff** [comment]
log is totally empty or I would send!!!

**Comment by macjeff** [comment]
Hours after I added it, it showed up in the ISY.   After waiting an hour for the dashboard to load I was able to do the AUTH.   It took almost an hour for devices to come in and status.  The blue message to get auth code would not go away even after I saw the green pop up that auth code was received.  About 20 min later it went away.

So I think the issue is not RING but since others are reporting the same types of issues with other nodeservers on PGC I think it is being overloaded either by a DOS attack or some kind of backed up queue issue
